src/screens/SignInV2/elements/BottomOverlay/index.js

```diff
@@ -47,9 +47,9 @@ const BottomOverlay = ({count, handleLogin, index, isLogin, title, onNextSlide,
               </Text>
               {',\n'}
               {'humanID verifies your humanity'}
-              <Text style={bottomOverlayStyles.descBold}>{' without storing\n'}</Text>
+              <Text>{' without storing\n'}</Text>
               {'your data or'}
-              <Text style={bottomOverlayStyles.descBold}>{' sharing'}</Text>
+              <Text>{' sharing'}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new changes remove the `bottomOverlayStyles.descBold` style from the Text components. If this was intentional and the bold styling is no longer needed, then it's fine. However, if the bold styling was removed by mistake, you should add it back.
```diff
- <Text>{' without storing\n'}</Text>
- {'your data or'}
- <Text>{' sharing'}</Text>
+ <Text style={bottomOverlayStyles.descBold}>{' without storing\n'}</Text>
+ {'your data or'}
+ <Text style={bottomOverlayStyles.descBold}>{' sharing'}</Text>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->